platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/RosterEntryNotFoundException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.roster;
+
+/**
+ * An exception thrown when a RosterEntry cannot be found, e.g. when searching by NodeId.
+ */
+public class RosterEntryNotFoundException extends RuntimeException {
+    /**
+     * A default constructor.
+     * @param message a message
+     */
+    public RosterEntryNotFoundException(String message) {
```

Comment:
nit: `message` is missing an annotation, and `final`

Reply:
Added the annotation and `final`.